src/App.css

```diff
@@ -1,26 +1,54 @@
 :root {
-  --light-font-color: #fff;
-  --bg-font-color: rgb(21 22 27);
-  --bg-color: #15161B;
-  --main-text-color-light: #000000;
-  --main-background-color-light: #ffffff;
-  --main-text-color-dark: #ffffff;
-  --main-background-color-dark: #000000;
-}
+  /* Dark theme colors: */
+  --background-color-dt: #15161b;
+  --primary-text-color-dt: #ffffff;
+  --primary-element-color-dt: #000000;
+  --secondary-text-color-dt: #cdcdcd;
+  --secondary-element-color-dt: rgba(0, 0, 0, 0.5); /* 50% opacity */
```

Comment:
Please, don't provide such a dump comments :)
We all are grown enough to understand Red-Green-Blue-Alpha notation, where Alpha = opacity.

Reply:
yes sir